cli/script/command-executor.ts

```diff
@@ -224,6 +259,54 @@ function appRename(command: cli.IAppRenameCommand): Promise<void> {
         });
 }
 
+function addCollaborator(command: cli.ICollaboratorAddCommand): Promise<void> {
+    throwForInvalidEmail(command.email);
+
+    return getAppId(command.appName)
+        .then((appId: string): Promise<void> => {
+            throwForInvalidAppId(appId, command.appName);
+
+            return sdk.addCollaborator(appId, command.email)
+                .then((): void => {
+                    log("Successfully added the \"" + command.email + "\" as a collaborator to \"" + command.appName +"\" app.");
+                });
+        });
+}
+
+function listCollaborators(command: cli.ICollaboratorListCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
```

Comment:
Yeah.. for readability I feel having new line makes sense.. besides its more for the sake of having return on a new line, so that its easy to find at a glance where the return is being handled. We have same thing in most of the places, check out appRename function above.